index.html

```diff
@@ -226,6 +224,7 @@ <h1 class="cabin_lovers">
     </div>
   </footer>
 </div>
+</div>
```

Comment:
why? 🧐

Reply:
check indent rainbow colors before you make a commit!

Reply:
och you are right :) first one is a page wrapper closure but the second one do nthg. Ok fixed now.